flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -496,22 +389,69 @@ static int32_t vtol_follower_control_accel(float dT)
 	return 0;
 }
 
+
+static float vtol_follower_control_altitude(float downCommand) {
```

Comment:
Yah.  I want to eliminate a lot of this copy and paste but often I end up just moving it around in preparation for that.